Kuroba/app/src/main/java/com/github/adamantcheese/chan/report/ReportActivityView.kt

```diff
@@ -0,0 +1,4 @@
+package com.github.adamantcheese.chan.report
```

Comment:
This interface is entirely useless and unused anyways.

Reply:
It exists just to satisfy the contract. Maybe something will be added there in the future, maybe not. Oh and it should be renamed, because there is no ReportActivity no more, but ReportController.

Reply:
Oh wait, yeah, it wasn't use. Now it is. It's still empty, though.